api/environments.md

```diff
@@ -11,23 +11,53 @@ Tradecloud has 3 environments available for customers:
 
 ## Production environment
 
-There is one [production environment](https://api.tradecloud1.com/) a.k.a. LIVE environment. Use [https://api.tradecloud1.com/v2/api-connector/](https://api.tradecloud1.com/v2/api-connector/) to send order \(responses\).
-
+There is one production environment a.k.a. LIVE environment. 
 The latest features will be made available by the Tradecloud continuous delivery process.
 
 It has an availability Service Level Objective of 95% per month.
 
-## Planned production maintenance
+#### Documentation
+
+You can use the endpoints below to integrate with the production environment. 
+Use [https://api.tradecloud1.com/v2/api-connector/](https://swagger-ui.prod.tradecloud1.com/?url=https://api.tradecloud1.com/v2/api-connector/specs.yaml) to send an order (response).
+
+You can [use the Swagger UI](tools/swagger-ui.md) to explore and test the Tradecloud API:
+
+| Service | Docs |
+|---|---|
+| https://api.tradecloud1.com/v2/api-connector | [YAML](https://api.tradecloud1.com/v2/api-connector/specs.yaml) / [Swagger UI](https://swagger-ui.prod.tradecloud1.com/?url=https://api.tradecloud1.com/v2/api-connector/specs.yaml) |
+| https://api.tradecloud1.com/v2/authentication | [YAML](https://api.tradecloud1.com/v2/authentication/specs.yaml) / [Swagger UI](https://swagger-ui.prod.tradecloud1.com/?url=https://api.tradecloud1.com/v2/authentication/specs.yaml) |
+| https://api.tradecloud1.com/v2/object-storage | [YAML](https://api.tradecloud1.com/v2/object-storage/specs.yaml) / [Swagger UI](https://swagger-ui.prod.tradecloud1.com/?url=https://api.tradecloud1.com/v2/object-storage/specs.yaml) |
+| https://api.tradecloud1.com/v2/order-search | [YAML](https://api.tradecloud1.com/v2/order-search/specs.yaml) / [Swagger UI](https://swagger-ui.prod.tradecloud1.com/?url=https://api.tradecloud1.com/v2/order-search/specs.yaml) |
```

Comment:
Please add `https://api.tradecloud1.com/v2/order/specs.yaml` 
(https://tradecloud.atlassian.net/browse/TC-7342 in progress)

Reply:
Added